packages/rendermime/src/renderers.ts

```diff
@@ -796,6 +796,19 @@ export function renderText(options: renderText.IRenderOptions): Promise<void> {
   return Promise.resolve(undefined);
 }
 
+/**
+ * Sanitize HTML out using native browser sanitizer.
+ *
+ * Compared to the `ISanitizer.sanitize` this does not allow to selectively
+ * allow to keep certain tags but escapes everything; on the other hand
+ * it is much faster as it uses platform-optimized code.
+ */
+function nativeSanitize(source: string): string {
+  const el = document.createElement('span');
+  el.textContent = source;
+  return el.innerHTML;
+}
```

Comment:
This is the recommended method of text sanitization as per [OWASP](https://cheatsheetseries.owasp.org/cheatsheets/DOM_based_XSS_Prevention_Cheat_Sheet.html#rule-6-populate-the-dom-using-safe-javascript-functions-or-properties)

We also already use it in other performance-sensitive places like in the completer:

https://github.com/jupyterlab/jupyterlab/blob/c9700a2802dabfb44109b4327473b6a593e7cc97/packages/completer/src/model.ts#L10-L17